src/app/core/components/footer/footer.component.html

```diff
@@ -1,4 +1,17 @@
-<footer class="govuk-footer" role="contentinfo">
+ <footer role="contentinfo">
+  <svg class="small-crown" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64 64" width="64" height="64">
```

Comment:
```suggestion
  <svg class="small-crown" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64 64" width="32" height="30">
```

This should be inside the govuk-width-container so it looks correct on larger viewports.